### DISCUSSION TITLE
Don't manage filename, only metadata

### DISCUSSION THREAD

**Discussion by Flecart** [davish/obsidian-full-calendar#473] [Feature Requests]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/davish/obsidian-full-calendar/discussions/categories/feature-requests) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/davish/obsidian-full-calendar/issues) didn't find any similar feature requests.

### 🔖 Feature description

I'd like a settings toggle where you can set if you want the plugin to manage the filenames of the events.

### ✔️ Solution

A toggle that says if the plugin should also manage the filename.
Currently it does it by default, but i don't see any reasons, other than efficiency, of why you would need the plugin to also manage the file name.

We should give users the choice!

**Comment by umamiMike** [comment]
I agree with this.  It seems like something that could be fully tracked by frontmatter  (file properties?) just like the start and end time and does not need to be in the title.  Thanks!

**Comment by jpfieber** [comment]
Agreed. Currently the filename is just the date and the title, both of which are available in the frontmatter. Manage those both, and let the filename be whatever it wants to be.

**Comment by ItsKazBorn** [comment]
Hard agree! I tried changing the filename and the event just disappeared from the calendar with no way to get it back.

I would also think that notes with all the fontmatter properties needed to show in the calendar should show in the calendar. 
So if I already have a note with content in it, I can just add the metadata needed (maybe add a button in the note options menu to add everything required?) and it should show up in the calendar automatically. The current option is creating a new note through the calendar and copy/pasting the content and other properties.
